<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tour</class>
 <widget class="QDialog" name="Tour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QTableWidget {
	background: transparent;
	border: none;
	font: helvetica;
	font-weight: bold;
}

QHeaderView::section
{
	spacing: 10px;
	background-color: gray;
	color: white;
	border: 1px solid ;
	margin: 1px;
	text-align: right;
	font-family: arial;
	font-size:12px;
}

QPushButton {
	background: darkgray;
	color: white;
	border-radius: 10px;
	height: 25px;
	width: 100px;
}

QPushButton:hover {
	color: black;
	background: lightblue;
}

QLabel {
	font: 50px;
	color: grey;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_back">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	width: 55px;
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="button_laTrip">
             <property name="styleSheet">
              <string notr="true">QPushButton {
	width: 200px;
}</string>
             </property>
             <property name="text">
              <string>Los Angeles Memorial Tour</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_fordTrip">
             <property name="text">
              <string>Ford Field Tour</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_customOrder">
             <property name="text">
              <string>Custom Order</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="button_startTour">
           <property name="text">
            <string>Most Efficient Tour</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="table_selectedTeams">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="table_allTeams">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_add">
             <property name="styleSheet">
              <string notr="true">QPushButton {
	width: 55px;
}</string>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButton_delete">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_7"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>0</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Los Anegels Tour</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>590</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Back to Main Page</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_laStartTour">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>570</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Tour</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="table_laTour">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>81</y>
         <width>901</width>
         <height>451</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget {
	color:  #413939;
	font: 16px;
	font-weight: bold;
}</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
